sphinx/api/types.rst

```diff
@@ -271,18 +270,18 @@ data. Air density at the 58th model level is calculated using temperature data e
 of pressure at that level using the ideal gas law. Pressure at the 58th model level is extrapolated from surface pressure
 data provided in this dataset using the hypsometric equation.
 
-==================== =======================================   ==================================
- Field Name           Data Type (ReanalysisMetaData.dtypes)     Units (ReanalysisMetaData.units)
-==================== =======================================   ==================================
- time                 datetime64[ns]                            datetime64[ns]
- windspeed            float                                     m/s
- windspeed_u          float                                     m/s
- windspeed_v          float                                     m/s
- wind_direction       float                                     degrees
- temperature          float                                     Kelvin
- density              float                                     kg/m^3
- surface_pressure     float                                     Pa
-==================== =======================================   ==================================
+==================== ================== ======================================= ==================================
+ Field Name           Descriptive Name   Data Type (ReanalysisMetaData.dtypes)   Units (ReanalysisMetaData.units)
+==================== ================== ======================================= ==================================
+ time                 time stamp         datetime64[ns]                          datetime64[ns]
+ WMETR_HorWdSpd       windspeed          float                                   m/s
+ WMETR_HorWdSpdU      windspeed-u        float                                   m/s
+ WMETR_HorWdSpdV      windspeed-v        float                                   m/s
```

Comment:
to be more descriptive, could say "northward wind speed component"

Reply:
This is updated in the recent wave of commits.